scripts/copper_bulk_submit.py

```diff
@@ -0,0 +1,45 @@
+import subprocess
+
+
+def grant_execute_permissions(directory_path) -> None:
+    """Grant execute permissions to all files in the given directory."""
+    # Construct the chmod command
+    command = ["chmod", "-R", "+x", directory_path]
+
+    try:
+        # Run the command
+        result = subprocess.run(command, check=True, text=True, capture_output=True)
+
+        # Print the command output
+        print("Permission command output:", result.stdout)
+    except subprocess.CalledProcessError as e:
+        # Print the error message
+        print("Error occurred while setting permissions:", e.stderr)
+
+
+def run_scp_command(local_path, remote_path) -> None:
```

Comment:
split remote path into remote_path and username. We can probably keep this cambridge HPC specific (ie login.hpc.cam.ac.uk hardcoded

Reply:
I have changed is to
```shell
default_username = "default_user"
    username = input(f"Enter your username (default: {default_username}): ").strip()
    
    # Use default_username if no input is provided
    if not username:
        username = default_username
        print(f"No username entered. Using default username: {username}")
```

Reply:
I just saw the comments up. I will modify this

Reply:
Yeah that works - I'm not sure the default username makes sense in this case, just always rely on input()